index.js

```diff
@@ -3,8 +3,8 @@ import {
     NativeModules
 } from 'react-native'
 
-const { Spokestack } = NativeModules
-const spokestackEmitter = new NativeEventEmitter(Spokestack)
+const { RNSpokestack } = NativeModules
+const spokestackEmitter = new NativeEventEmitter(RNSpokestack)
```

Comment:
cool...ios/RNSpokestack.m has `@implementation RNSpokestack` so 👍

Reply:
just to point out you're looking at the outdated version.